README.md

```diff
@@ -1,61 +1,407 @@
 
-The `argo_nginx.conf` is needed but should not be changed.
-All configuration is done via an `.env` file:
+4. You can adjust values for your needs (look at `.env.demo` file that describe configurations).
 
 ```env
 #file checker API configuration
 FILECHECKERAPI_IMAGE=ghcr.io/oneargo/argoformatchecker/python-api
 FILECHECKERAPI_IMAGE_TAG=latest
-FILECHECKER_ENABLED=enabled #use 'enabled' or 'disabled' to enable/disable the service
-FILECHECKER_PORT=8000
 
 #decoder API configuration
 ARGODECODERAPI_IMAGE=boilerplateapi # to change with decoder image
 ARGODECODERAPI_IMAGE_TAG=latest
-DECODER_ENABLED=enabled #use 'enabled' or 'disabled' to enable/disable the service
-DECODER_PORT=8001
+DECODER_USER_UID=1000
+DECODER_USER_GID=1000
+
+#jupyterlab configuration :
+JUPYTERLAB_IMAGE=quay.io/jupyter/datascience-notebook
+JUPYTERLAB_IMAGE_TAG=python-3.13
+JUPYTER_USER_UID=1000
 
 #webserveur NGINX configuration
-WEBSERVER_ENABLED=enabled #use 'enabled' or 'disabled' to enable/disable the service
+WEBSERVER_HOST="127.0.0.1"
 WEBSERVER_PORT=8080
 
-#default profile to use with docker compose up.
-COMPOSE_PROFILES=enabled #  DO NOT CHANGE
+#Configure which service to enable (decoder,checker,jupyter) separated by a comma. will be run with 'docker compose up'
+COMPOSE_PROFILES=decoder,checker,jupyter
+```
+
+Webserver will always be enabled as it is necessary to access others services.
+
+Warning : An issue prevents the web server from starting if the Jupyter service is not running. For the moment keep `jupyter` in the list of services in `COMPOSE_PROFILES`.
+
+The `argo_nginx.conf.template` file is needed and **must not** be changed.
+
+---
+
+### ▶ Launch Services (via Docker Desktop – No Terminal Needed)
+
+All services can be launched directly from **Docker Desktop**, without using the command line.
+
+---
+
+1. Open Docker Desktop
+
+   Launch Docker Desktop and wait until it shows:
+
+   ```text
+   Docker Desktop — Running
+   ```
+
+2. Open the _Containers_ view
+
+   In the left sidebar, click:
+
+   **Containers → [your-repo-name]**
+
+   You should see a list of services defined in `compose.yaml`.
+
+3. Start all services
+
+   Next to the project name, click:
+
+   👉 **▶ Start**
+
+   Docker Desktop will:
+
+   - pull the container images (first time only)
+   - start all services listed in `compose.yaml`
+   - show green “Running” indicators
+
+4. Check that everything is running
+
+   Each service should display:
+
+   - a **green dot**
+   - a **Running** status
+
+   You can click any container to view logs, ports, and details.
+
+5. Stop all services
+
+   To stop everything safely, click:
+
+   👉 **⏹ Stop**
+
+6. Restart if needed
+
+   You can restart the entire stack by clicking:
+
+   👉 **⟳ Restart**
+
+#### 💡 Optional: Command Line (for advanced users)
+
+##### 🚀 Launch services
+
+If you prefer using a terminal:
+
+```bash
+docker compose up -d
+docker compose ps
```

Comment:
Would it be useful to tell or show what the user should expect. for example see a list of 4 containers along witheir status, port, names etc

<img width="614" height="136" alt="image" src="https://github.com/user-attachments/assets/f200e15b-b271-4924-a9d7-4b852c174f32" />